### PR TITLE
M3 192/enable multiple fastq inputs

### PR DESCRIPTION
**Pull request by doron-st**
A common input pattern is a sub directory per sample, 
and multiple fastq file pairs inside. (each file pair represents a READ-GROUP)
extend the pipeline to accept such input pattern

### CODE REVIEW

File: nf/maestro3/genotype_fastq.nf [hunk 1]
```diff
@@ -193,35 +224,36 @@ process mergeAlignedAndUnalignedBam {
...
     output:
-    set val(id), file("${flowcell_bam}.nodup.bam") into nodup_bam_ch
+    set val(id), file("${id}.nodup.bam") into nodup_bam_ch
 
     //Task is assuming query-sorted input so that the Secondary and Supplementary reads get marked correctly.
     //This works because the output of BWA is query-grouped and therefore, so is the output of MergeBamAlignment.
     //While query-grouped isn't actually query-sorted, it's good enough for MarkDuplicates with ASSUME_SORT_ORDER="queryname"
     script:
     maxJVMMem = task.memory.toGiga() -1
+    inputs_list = read_group_bams.join(" --INPUT ")
```
Comment (avigailmo) [inline nf/maestro3/genotype_fastq.nf]: what does this mean?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ curl -s https://get.nextflow.io | bash
 ### Maestro3
 - Reference based Genotyping-by-sequencing (GBS) - Based on BWA and HaplotypeRecaller
 
+### Samples Inputs
+- --fastq_dir (single file per sample) - samples with single pair of fastq files can be stored directly under params.fastq_dir<br>
```
[the remark this answers is not in the document]
Comment (doron-st) [reply README.md]: I don't understand. The "-" in the begining of the line makes a bulleted list. Is that not what you meant?

File: config/nextflow.aws.config [hunk 3]
```diff
@@ -1,7 +1,7 @@
 executor.name = 'awsbatch'
 
 process{
-  queue = 'GenomicsDefaultQueue-db299dfc1fe10b9'
+  queue = 'maestro3'
```
[the remark this answers is not in the document]
Comment (doron-st) [reply config/nextflow.aws.config]: yes. It was quite easy to make. 
I was trying to add aws tags to the machines initiated by maestro3 jobs.
On the way also expended the queue to accept 0 -1000 vCPUs.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (avigailmo) [reply README.md]: yes, this is what I meant :)

File: nf/maestro3/genotype_fastq.nf (hunk 1 above)
Comment (doron-st) [reply nf/maestro3/genotype_fastq.nf]: The tool accepts multiple inputs via CL like this:
--INPUT <input1> --INPUT <input2> ...
The join with (" --INPUT ") separator on the list of file names creates this string.

**Pull request merged**